### PR TITLE
added footer with just links & social icons

### PR DESCRIPTION
**Pull request by haqcked**
the design is bare, i tried adding the illustrations to design it a bit but i'm having trouble with positioning them. I spent too much time but ended up with this very simple one 😅.

For some reason, fontawesome icons do not show up when i add them. I tried ionicons.io, included the script in application.html.erb. Im not sure if we can keep the ionicon icons instead of fontawesome ones.

need suggestions

Screen Shot 2023-05-13 at 10 22 05 PM

### CODE REVIEW

**Comment by wJoenn** [comment]
fontaweome's brand icons are not being imported in the application yet. That's why you can't use them.
Try adding the brand icons to the application.scss and it should work

Isn't the copyright thing a legal thing ? Can we just write it and don't care about it ?
Chloe doesn't have a twitter account not does she want one so you can remove that :) 
I would also change the navbar to be horizontal and centered maybe ?

It'd be easier to have a design on figma first to able to quickly make changes

**Comment by haqcked** [comment]
> 

I see. I thought the existing @imports are enough, ive added the 'brands'. now it works. Ive deleted the ionicons.

About the copyright stuff, im sure we can all agree on that. Ive included it mainly because i needed more stuff to populate the footer 😅

I will try the horizontal navbar suggestion. I think it would look better.

Ive actually attempted a blueprint on figma but even there I didnt know what else to add.

**Comment by haqcked** [comment]
I added one plant on the bottom right to fill the empty space 😅

what do you guys think?

Screen Shot 2023-05-15 at 10 34 32 PM

**Comment by wJoenn** [comment]
The logo is being cut by the svg. You can fix that with z-index

**Comment by haqcked** [comment]
> The logo is being cut by the svg. You can fix that with z-index

oh yeah, didnt notice that. on it.

**Comment by haqcked** [comment]
I didnt add the z-index, I fixed the wave background image instead. 

Screen Shot 2023-05-16 at 12 01 24 AM

**Review by k-kenney — APPROVED** [review]
I think it looks fantastic. Louis gave you some helpful tips and now it looks clean and nice. I like the leaf you added to continue with the theme. Great work on the footer, @haqcked !

**Comment by StephanK83** [comment]
Hey guys...Some things here. I'm tooo lazy to pull this branch right now. Could you call this page through your phone and send us a screenshot? Would be happy to see, how logo, wave and plant behave. I guess, wave shouldn't be squeezed and plant shouldn't be shown at all in the mobile version. 

Also everything in the footer, what is right to the logo should be reararranged below the logo in the mobile/ smaller screen size version? 

What do you think dearest people?

File: app/frontend/stylesheets/pages/_index.scss
Comment (wJoenn) [inline app/frontend/stylesheets/pages/_index.scss:4]: Can you move that up between the calendar and membership please

File: app/views/pages/home.html.erb
Comment (wJoenn) [inline app/views/pages/home.html.erb:12]: Indentation isn't correct

File: app/views/pages/partials/_footer.html.erb
Comment (wJoenn) [inline app/views/pages/partials/_footer.html.erb:15]: A list of links should be a `<nav>`, you also don't need to nest that inside a div, just give the content-links class to the nav directly

File: app/views/pages/partials/_footer.html.erb
Comment (wJoenn) [inline app/views/pages/partials/_footer.html.erb:19]: There's no reason to nest the h4 inside a div

File: app/frontend/stylesheets/pages/partials/_footer.scss
Comment (wJoenn) [inline app/frontend/stylesheets/pages/partials/_footer.scss:99]: Use SCSS nested elements please

File: app/frontend/stylesheets/pages/partials/_footer.scss
Comment (wJoenn) [inline app/frontend/stylesheets/pages/partials/_footer.scss:103]: You already have a footer-container selector

**Review by wJoenn — CHANGES_REQUESTED** [review]
I mentionned it in a comment but the footer is currently veeeeeeery big.
Can you fix that ? Also add the content of your footer inside a .container please

File: app/frontend/stylesheets/pages/_index.scss
Comment (haqcked) [reply app/frontend/stylesheets/pages/_index.scss:4]: done

File: app/views/pages/home.html.erb
Comment (haqcked) [reply app/views/pages/home.html.erb:12]: corrected

File: app/views/pages/partials/_footer.html.erb
Comment (haqcked) [reply app/views/pages/partials/_footer.html.erb:15]: done

File: app/views/pages/partials/_footer.html.erb
Comment (haqcked) [reply app/views/pages/partials/_footer.html.erb:7]: got it.

File: app/views/pages/partials/_footer.html.erb
Comment (haqcked) [reply app/views/pages/partials/_footer.html.erb:27]: ok

File: app/frontend/stylesheets/pages/partials/_footer.scss
Comment (haqcked) [reply app/frontend/stylesheets/pages/partials/_footer.scss:4]: deleted

File: app/frontend/stylesheets/pages/partials/_footer.scss
Comment (haqcked) [reply app/frontend/stylesheets/pages/partials/_footer.scss:24]: the reason for this was the wave image was taking most of the upper space of the footer. The image is also half transparent and half the wave. I needed add more height to the footer itself so there is enough space for the other elements.

in the revised version, ive changed approach by putting the wave image outside and above the footer. set it to overlap behind the footer. the footer height is just now set to 35vh + the wave on top of the footer

Im not very satisfied with my solution, i think there is a better and easier way to position the wave in the footer. This has been very challenging for me. perhaps someone can suggest on how to do it better

Comment (haqcked) [reply app/frontend/stylesheets/pages/partials/_footer.scss:24]: Screen Shot 2023-05-20 at 6 54 09 PM

File: app/frontend/stylesheets/pages/partials/_footer.scss
Comment (haqcked) [reply app/frontend/stylesheets/pages/partials/_footer.scss:36]: thanks for this suggestion. I realized i didnt need to provide a grid (space) for the images. I can just position them anywhere.

File: app/frontend/stylesheets/pages/partials/_footer.scss
Comment (haqcked) [reply app/frontend/stylesheets/pages/partials/_footer.scss:60]: Im struggling to wrap my head around the positions in css, ive still given each of the images the position absolute because if not they wont stay on the position where i want them to be. Perhaps you can help me with this.

File: app/frontend/stylesheets/pages/partials/_footer.scss
Comment (haqcked) [reply app/frontend/stylesheets/pages/partials/_footer.scss:99]: i think the revision is better now

File: app/frontend/stylesheets/pages/partials/_footer.scss
Comment (haqcked) [reply app/frontend/stylesheets/pages/partials/_footer.scss:103]: got it

File: app/frontend/stylesheets/pages/partials/_footer.scss
Comment (haqcked) [reply app/frontend/stylesheets/pages/partials/_footer.scss:110]: yup

**Comment by haqcked** [comment]
Screen Shot 2023-05-20 at 7 05 21 PM

File: app/views/pages/partials/_footer.html.erb
Comment (wJoenn) [inline app/views/pages/partials/_footer.html.erb:31]: The wave should be inside the footer.
You can also remove the empty spaces between parent elements and child elements. The indentation makes it readable enough :) That means line 4, 10, 25 and 30. The rest of your spacing is nice.

Your nav should not have a ul or li
It nav with anchors tags directly inside
```
nav
  a
  a
  a
/nav
```

Also the video page will not be done before a while so you can remove this for now.

**Review by wJoenn — CHANGES_REQUESTED** [review]
Terribly sorry about the delay @haqcked , I didn't realize you made those changes at all.
Please make sure to change your tag to "waiting to be reviewed" after updated your PR and spam me with messages if I don't reply within a couple hours, that usually means I forgot about it or I didn't notice the notification

File: app/views/pages/partials/_footer.html.erb
Comment (haqcked) [reply app/views/pages/partials/_footer.html.erb:31]: all done

**Comment by wJoenn** [comment]
Your wave is not visible in 1440p
image

and not attached in 720p The images are also broken in this resolution
image

Please make this reponsive for 720p to 1440p screens

**Comment by wJoenn** [comment]
Remove the absolute position on the wave and let it just sit on top of the rest like one div above another.
Also there is only one media query for this. Your component should not need a single proprety inside a `@media query` for the 720p to 1440p scale

**Comment by haqcked** [comment]
the gap prob has been eliminated. 

The footer looks ok at 720p:
Screenshot 2023-05-27 at 6 24 48 PM


but at 1440p the elements are so spaced out:
Screenshot 2023-05-27 at 6 25 12 PM


I need suggestions pls

**Comment by wJoenn** [comment]
@haqcked try using a div.container to center the content like we do with the rest of the website

**Comment by wJoenn** [comment]
Btw the way I intended the tag "waiting for review" and "require changes" is a poster set his PR as waiting for review when he thinks it's ready to be merged and a reviewed changes the tag to requires changes when the PR is in need of changed before the merge 🙂

**Comment by haqcked** [comment]
720p
Screenshot 2023-05-30 at 3 55 14 PM

1440p
Screenshot 2023-05-30 at 3 55 00 PM

What do you think?

**Comment by haqcked** [comment]
https://github.com/wJoenn/YogaUnalome/assets/114689012/1dd73044-1992-4b22-b04d-80d0660caffb

**Review by wJoenn — APPROVED** [review]

**Pull request merged**
